src/MockPlayerBridge.ts

```diff
@@ -41,6 +76,85 @@ export class MockPlayerBridge extends AbstractPlayerBridge {
     return 'dummy-stream-id';
   }
 
+  protected pause() {
+    this.playbackState = 'PAUSED';
+    document.dispatchEvent(new Event('livery-playback-change'));
+  }
+
+  protected play() {
+    this.playbackState = 'PLAYING';
+    document.dispatchEvent(new Event('livery-playback-change'));
+  }
+
+  protected reload() {
+    this.playbackState = 'PLAYING';
+    document.dispatchEvent(new Event('livery-playback-change'));
+  }
+
+  protected seek(position: number) {
+    this.playbackState = 'SEEKING';
+    document.dispatchEvent(new Event('livery-playback-change'));
+    return position;
+  }
+
+  protected selectQuality(index: number) {
+    return index;
+  }
+
+  protected setControlsDisabled(disabled: boolean) {
+    return disabled;
+  }
+
+  protected setDisplay(display: DisplayMode) {
+    this.display = display;
+    document.dispatchEvent(new Event('livery-display-change'));
+    return display;
+  }
+
+  protected setMuted(muted: boolean) {
+    this.muted = muted;
+    document.dispatchEvent(new Event('livery-muted-change'));
+    return muted;
+  }
+
+  protected submitUserFeedback(payload: UserFeedbackPayload) {
+    return payload;
+  }
+
+  protected subscribeConfig(listener: (value: Config) => void) {
+    const config: Config = {
+      controls: {
+        cast: false,
```

Comment:
I would default these controls to true instead of false.

Or as a nice to have you could add inputs to this element to be able to control these mocked values for testing purposes.